model/testing/src/icon4py/model/testing/helpers.py

```diff
@@ -44,6 +45,10 @@ def is_python(backend) -> bool:
     return is_embedded(backend) or is_roundtrip(backend)
 
 
+def is_dace(backend) -> bool:
```

Comment:
Please add type annotations to all `is_<backend>`, because they will highlight problems, see (1)